code/game/objects/items/salvage.dm

```diff
@@ -75,7 +75,7 @@
 /// Loot salvage, gained from fighting space simplemobs.
 
 /obj/item/salvage/loot
-	desc = "A tonne of salvage looted from a fallen foe. Who spawned the base type? Report this on the github."
+	desc = "If you can see this forbidden salvage, report it on GitHub."
```

Comment:
aww, I kinda like this one

Reply:
My main concern was that it's very verbose for something that you do not want to appear, ever.

Priority is communicating with the player that they should report, and that shouldn't be diluted IMHO.